optuna/study.py

```diff
@@ -239,7 +240,7 @@ def optimize(
             catch:
                 A study continues to run even when a trial raises one of the exceptions specified
                 in this argument. Default is an empty tuple, i.e. the study will stop for any
-                exception except for :class:`~structs.TrialPruned`.
+                exception except for :class:`~exceptions.TrialPruned`.
```

Comment:
The path of `TrialPruned` is needed to correct to generate a proper link (this is not a mistake introduced by this PR though):
```suggestion
                exception except for :class:`~optuna.exceptions.TrialPruned`.
```

Reply:
As for links, maybe this line too actually https://github.com/pfnet/optuna/pull/691/files#diff-8293551812da2037a53e2807b3337bceR27.

Reply:
Thanks. Both of the links are fixed.

optuna/study.py: https://github.com/pfnet/optuna/pull/691/commits/8096b88fc3c784ceae5519ab7605191c6fc39016
![image](https://user-images.githubusercontent.com/3255979/68737863-12ace280-0628-11ea-9bc2-5814b8536e2a.png)

optuna/structs.py: https://github.com/pfnet/optuna/pull/691/commits/f4ed8ae612f27deb72cc4164e9fe835908c7a114
![image](https://user-images.githubusercontent.com/3255979/68737885-23f5ef00-0628-11ea-9930-21b040672be7.png)